docs/DeveloperGuide.md

```diff
@@ -766,37 +782,51 @@ Priorities: 1 (must have), 2 (nice to have), 3 (unlikely to have)
 
 (For all use cases below, the **System** is the `tCheck` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use Case: UC01 - Archive a contact detail of an employee**
+**Use Case: UC01 - Archive an employee**
 
 **MSS**
 
-1. User chooses to archive one employee’s contact details. 
-2. tCheck requests for details of the employee to archive.
-3. User enters the archive details.
-4. tCheck will move this corresponding contact detail into the archive and displays a success message.
+1. User gives command to list all employees.
+2. User archives an employee from employee directory. 
+3. tCheck will move this corresponding employee into the archive and displays a success message.
      
      Use case ends.
 
 
 **Extensions**
 
-* 3a. tCheck detects an incorrect input format or is unable to find the entered data.
+* 2a. tCheck detects an incorrect input format.
         
-        * 3a1. tCheck requests for the correct data.
+        * 2a1. tCheck requests the user to re-enter with the correct format.
         
-    	* 3a2. User enters new data.
+    	* 2a2. User enters new data.
     	
-    	Steps 3a1-3a2 are repeated until the data entered are correct.
+    	Steps 2a1-2a2 are repeated until the data entered is in the correct format.
     	
-    	Use case resumes from step 4.
+    	Use case resumes from step 3.
 
+* 2b. tCheck detects that the specified employee does not exist.
+        
+        * 2b1. tCheck requests the user to re-enter a valid index that corresponds with an existing employee.
+        
+        * 2b2. User enters new index.
+        
+        Steps 2b1-2b2 are repeated until the index entered is a valid index.
```

Comment:
I think that this is a better phrasing: "until the index entered is valid"

Reply:
agree 👍